src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -29,10 +29,12 @@ public class DocumentExecuter : IDocumentExecuter
     {
         private readonly IDocumentBuilder _documentBuilder;
         private readonly IDocumentValidator _documentValidator;
+        public bool Strict { get; set; }
```

Comment:
I have a number of updates to the GraphQL Language in https://github.com/graphql-dotnet/graphql-dotnet/pull/65 to help support validation.  There are a number of validation scenarios that will be super hard to do until that is merged in.  If you are able to add your specific validation scenarios to https://github.com/graphql-dotnet/graphql-dotnet/issues/10 then we can double check that they aren't already solved with the core validation rules provided by the spec.